### DISCUSSION TITLE
Highest rate sensor

### DISCUSSION THREAD

**Discussion by asic79** [BottlecapDave/HomeAssistant-OctopusEnergy#497] [Ideas]
Hi,

Not sure if this has been proposed before; I couldn't find anything on a quick search. 

Is it possible to have a highest rates sensor functioning similarly to the lowest rate one?

One possible and quite neat usecase will be to switch to battery stored energy when electricity is the most expensive. 

Usually this is between 16:00 and 19:30 but the rates can also spike throughout the day. 

Thanks

**Comment by BottlecapDave** [comment]
Hi there. You can use the target rate sensors, but turn on the [invert](https://github.com/BottlecapDave/HomeAssistant-OctopusEnergy/blob/develop/_docs/setup_target_rate.md#invert-target-rates) behaviour. For import meters, this will find the most expensive rates.
